src/dug/core.py

```diff
@@ -616,12 +605,8 @@ class Dug:
     variables_index = "variables_index"
     kg_index = "kg_index"
 
-    def __init__(self):
-        self._search = Search(
-            host=cfg.elasticsearch_host,
-            port=cfg.elasticsearch_port,
-            indices=[self.concepts_index, self.variables_index, self.kg_index]
-        )
+    def __init__(self, cfg=config.from_env()):
+        self._search = Search(indices=[self.concepts_index, self.variables_index, self.kg_index])
```

Comment:
we don't do anything with the cfg param here, and I think it should be a required param as well (and the usages updated accordingly)

```suggestion
    def __init__(self, cfg: Config):
        self._cfg = cfg
        self._search = Search(cfg, indices=[self.concepts_index, self.variables_index, self.kg_index])
```